cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -81,6 +83,9 @@
   private NettyHttpService httpService;
   private volatile Thread runningThread;
   private final Map<String, WorkflowActionNode> status = new ConcurrentHashMap<String, WorkflowActionNode>();
+  private volatile boolean isSuspended;
```

Comment:
By common Java convention, variable name don't have `is` (otherwise the getter method for the variable will become `isIsSuspended`. Just call it `suspended`.